app/views/items/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Item Data</h1>
+
+<p>Name: <%= @item[:data][:attributes][:name] %></p>
+<p>Description: <%= @item[:data][:attributes][:description] %></p>
```

Comment:
Creating a PORO for `Item` would allow you to access the data via attr_reader methods similar to how you have it set up for merchants.